tests/test_ind_indygo_finance.py

```diff
@@ -40,29 +40,26 @@ def test_description():
 
 
 def test_start():
-    assert parsed_items[0]["start"] == datetime(2023, 1, 19, 8, 30)
+    assert parsed_items[0]["start"] == datetime(2024, 1, 18, 8, 30)
 
 
 # def test_end():
 #     assert parsed_items[0]["end"] == datetime(2019, 1, 1, 0, 0)
 
 
 def test_time_notes():
-    assert (
-        parsed_items[0]["time_notes"]
-        == "Board meetings are set for 8:30AM unless otherwise noted in meeting description. Please double check the website before the meeting date."  # noqa
-    )
+    assert parsed_items[0]["time_notes"] == ""  # noqa
 
 
 def test_id():
     assert (
         parsed_items[0]["id"]
-        == "ind_indygo_finance/202301190830/x/indygo_finance_committee"  # noqa
+        == "ind_indygo_finance/202401180830/x/indygo_finance_committee"  # noqa
     )
 
 
 def test_status():
-    assert parsed_items[0]["status"] == "passed"
+    assert parsed_items[0]["status"] == "tentative"
```

Comment:
See this [comment](https://github.com/City-Bureau/city-scrapers-indianapolis/pull/14/files#r1463715174).